scripts/no_addons.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 Elico Corp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+import re
+from .addons import ODOO_CONF, REGEX_ADDONS_PATH
+
+DEFAULT_ADDONS_PATH = 'addons_path = /opt/odoo/sources/odoo/addons'
+
+
+def main():
```

Comment:
I know it's already mentioned somewhere else, but can you mention in a comment here the lines of bash code that were not working?